app/helpers/admin_helper.rb

```diff
@@ -10,4 +10,14 @@ def service_sp_component_url(service)
   def service_msa_component_url(service)
     service.msa_component_id.nil? ? 'No MSA configured' : (link_to service.msa_component_id, msa_component_path(service.msa_component_id))
   end
+
+  def get_all_emails_for_group_of_teams(teams)
```

Comment:
How about
`  
  def get_all_emails_for_group_of_teams(teams)
    email_addresses = teams&.each { |team|
      get_users_in_group(group_name: team.team_alias).map { |cognito_user| as_team_member(cognito_user: cognito_user) }
    }
    &.flatten(&:email) || []

    email_addresses <<  t('user_journey.confirmation.support_email')
  end
`

Reply:
Annoyingly i tried something like this before but it just kept returning a list of ID's. I will look again and trying it this way

Reply:
Ummh, it works for me, maybe there is difference btw what you tried and this in any case
 why not use `team_recipients` so this could be

`
  def get_all_emails_for_group_of_teams(teams)
    email_addresses = teams && teams.map { |team| team_recipients(team.team_alias) }
                                                                         .flatten(&:email) || []

    email_addresses << t('user_journey.confirmation.support_email')
  end
`

Reply:
Let me try, using team_recipients sounds like a good idea

Reply:
Whats the purpose of the `team &&`. That isnt necessary is it?